packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -200,6 +201,92 @@ describe('autocomplete-js', () => {
     ).toHaveTextContent('No results template');
   });
 
+  test("doesn't render empty template on no query when openOnFocus is false", async () => {
+    const container = document.createElement('div');
+    const panelContainer = document.createElement('div');
+
+    document.body.appendChild(panelContainer);
+    autocomplete<{ label: string }>({
+      container,
+      panelContainer,
+      openOnFocus: false,
+      getSources() {
+        return [
+          {
+            getItems() {
+              return [];
+            },
+            templates: {
+              item({ item }) {
+                return item.label;
+              },
+              empty() {
+                return 'No results template';
+              },
+            },
+          },
+        ];
+      },
+    });
+
+    const input = container.querySelector<HTMLInputElement>('.aa-Input');
+
+    fireEvent.input(input, { target: { value: '' } });
+
+    await waitFor(() => {
```

Comment:
I realize now that there are `expect`s in all `waitFor` while they should just be `querySelector`.

Reply:
I think that's how they recommend to use it (cf: https://testing-library.com/docs/guide-disappearance/#2-using-waitfor), that being said, the second test should be removed 
e.g.: 
```js
    await waitFor(() => {
      expect(
        panelContainer.querySelector<HTMLElement>('.aa-Panel')
      ).not.toBeInTheDocument();
    });

    expect(
      panelContainer.querySelector<HTMLElement>('.aa-Panel')
    ).not.toBeInTheDocument();
```